swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

```diff
@@ -0,0 +1,312 @@
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: Add Org(s) completed.
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+
+  /internal/organizations/remove/org/{orgs}:
+    description: "Initiate removing a given list of Org IDs from database sync list"
+    parameters:
+      - name: orgs
+        in: path
+        required: true
+        schema:
+          type: string
+    put:
+      summary: "Remove a given list of organizations from the database sync list."
+      operationId: removeOrgsFromSyncList
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request for removing organizations from the sync list has completed successfully."
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: Remove Org(s) completed.
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+
+  /internal/organizations/search/org/{org_id}:
+    description: "Initiate a search to check if given Org ID is in sync list"
+    parameters:
+      - name: org_id
+        in: path
+        required: true
+        schema:
+          type: string
+    get:
+      summary: "Check if given Org ID is in the database sync list."
+      operationId: hasOrgInSyncList
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request to check the sync list for given org ID has successfully completed."
+          content:
+            text/plain:
+              schema:
+                type: boolean
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+
+  /internal/organizations/inventory/org/{org_id}:
+    description: "Initiate viewing conduit representation of an org's systems from RHSM"
+    parameters:
+      - name: org_id
+        in: path
+        required: true
+        schema:
+          type: string
+    get:
```

Comment:
The inventory list should be paged with some reasonable maximum number returned in a given call. 
You can see https://github.com/RedHatInsights/rhsm-subscriptions/blob/b64cc53272ed8f7bea37e19158bf9c50c7663f52/api/rhsm-subscriptions-api-spec.yaml#L675 & the offset & limit usage for details. 
The API should specify what values will be used by default & what the maximum values that can be specified for limit would be.